packages/jasmine/src/jasmine_node_test.bzl

```diff
@@ -35,6 +35,14 @@ def jasmine_node_test(
         **kwargs):
     """Runs tests in NodeJS using the Jasmine test runner.
 
+    Detailed XML test results are found in the standard `bazel-testlogs`
+    directory. This may be symlinked in your workspace.
+    See https://docs.bazel.build/versions/master/output_directories.html
+
+    If using self-managed npm dependencies, ensure that the `jasmine-reporters`
```

Comment:
nit: this is likely a very rare use case so may not be even worth mentioning as it may lead to some confusion; if we do mention it then `v8-coverage` is also required dep to use `jasmine_node_test` from `index.from_src.bzl`.

Reply:
Ok sounds good. Shall I remove this note then?

Reply:
I think so. Anyone doing self-managed npm deps is going down the less supported path for now anyway.